### ISSUE TITLE
An entry is added instead of completing input when entering Japanese (MS-IME)

### ISSUE THREAD

**Issue by AkMgc** [benrhughes/todotxt.net#417]
Hello, I use todotxt.net in Japanese environment on Windows 10.

So far Japanese characters are handled well on todotxt.net, except that
an entry is added when pressing Enter, instead of completing Japanese characters.

For example, when I enter "(A) +プロジェクト壱 はがきを購入する"
I expect to complete the first sequence of Japanese characters, "プロジェクト壱",
and then enter the following "はがきを購入する",
but when I press Enter after 壱 to complete the first Japanese sequences, 
the whole entry is added, before adding the following characters.

In short, sending "Enter" to Japanese IME is preceded and catched by todotxt.net.

It is very troublesome and it is rarely found in other softwares, so
I'm glad if this will be fixed.

Best wishes,

AkMgc
